### DISCUSSION TITLE
Supabase not working with Angular 17

### DISCUSSION THREAD

**Discussion by kaaai3** [supabase/supabase#19307] [Contribute to Supabase]
# Bug report

- [x] I confirm this is a bug with Supabase, not with my own application. 
- [x] I confirm I have searched the [Docs](https://docs.supabase.com), GitHub [Discussions](https://github.com/supabase/supabase/discussions), and [Discord](https://discord.supabase.com). 

## Describe the bug

Hey, I just created a new Angular 17 Application. After adding the latest supabase client the serve and build command will not work properly anymore. I was following this guide https://supabase.com/docs/guides/getting-started/tutorials/with-angular

## To Reproduce

Steps to reproduce the behavior, please provide code snippets or a repository:

1. fork this repo https://github.com/kaaai3/repro-supabase
2. serve the angular app
3. It will load infinite

## Expected behavior

For Angular to load with a created client.

## System information

- OS: Windows
- Version of supabase-js: 2.39.0
- Version of Node.js: 18.18.2

**Comment by saltcod** [comment]
Hey @kaaai3 — Discord is probably the best place to get help on this one. Going to transfer this to a Discussion.

**Comment by supersu-man** [comment]
I am also facing issues on Angular 17.
The project I've built using Angular 16 a month ago works just fine.
I have a raised an issue [here](https://github.com/supabase/supabase-js/issues/936) and [here](https://www.reddit.com/r/Supabase/comments/18xnvr0/acquiring_an_exclusive_navigator_lockmanager/).

Edit: Just moved back to Angular 16 to test, it doesn't work either.

**Reply by karimg06** [reply]
I am also facing issues on Angular 17 as soon as i use supabase the localhost page loads permanently.

**Reply by aaditya890** [reply]
where's the solution broo..?

**Reply by karimg06** [reply]
I couldn't solve it and just switched to another backend as a service..

**Comment by aaditya890** [comment]
Ok
